### ISSUE TITLE
nitro add throws error if web root is named 'html'

### ISSUE THREAD

**Issue by mach1media** [craftcms/nitro#271]
### Description

`nitro add` throws unrecoverable error if web root does not use one of the common names in docs, ex: `public`, `public_html`, or `web`.

```
sitename.local/
  craft/
  html/
```

### Steps to reproduce

1. Navigate to `sitename.local` existing project's root
2. Run `nitro add` and see the following output...

```
% nitro add
Adding site…
Enter the hostname [sitename.local]: sitename.local
  ✓ setting hostname to sitename.local
  ✓ adding site ~/sites/sitename.local
Error: unable to locate a webroot
```

3. Expecting prompt like `Enter the webroot for the site [web]:`

### Additional info

- Nitro version: 2.0.3
- Docker version: 3.1.0

**Comment by jasonmccallister** [comment]
That makes sense, we should not return an error but it might make sense to add that as part of the detection... Right now we support the following:

- web
- public
- public_html

I don't think its too much to add `html` but we should not throw an error if we can't find one automatically.

**Comment by jasonmccallister** [comment]
This was released with 2.0.4. Please let me know if this is still an issue but it should be resolved!
